wrappers/java/pom.xml

```diff
@@ -0,0 +1,59 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.ensembl.hive</groupId>
+    <artifactId>eHive</artifactId>
+    <version>1.0</version>
```

Comment:
What does this version represent ? There is already a version defined in src/main/java/org/ensembl/hive/BaseRunnable.java